lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -88,11 +88,14 @@ defmodule ExUnit.CLIFormatter do
     {:noreply, config}
   end
 
-  def handle_cast({:case_finished, %ExUnit.TestCase{state: {:failed, failures}} = test_case}, config) do
-    formatted = format_test_case_failure(test_case, failures, config.failure_counter + 1,
+  def handle_cast({:case_finished, %ExUnit.TestCase{name: case_name, state: {:failed, failures}} = test_case}, config) do
+    case_tests = case_name.__ex_unit__(:case).tests
```

Comment:
Oh, so you should be able to simply do: `length(test_case.tests)`. Sorry about the confusion, I had the old API in my mind.